packages/frontend/src/components/common/Table/types.ts

```diff
@@ -8,7 +8,10 @@ import { Cell, ColumnDef, Header } from '@tanstack/react-table';
 import { MouseEventHandler } from 'react';
 
 export type HeaderProps = { header: Header<ResultRow> };
-export type CellContextMenuProps = { cell: Cell<ResultRow> };
+export type CellContextMenuProps = {
+    isEditMode?: boolean;
```

Comment:
I would keep this type as it was since the table isn't aware or manages that prop. 
We could instead update the component type:
```CellContextMenu: React.FC<CellContextMenuProps & {isEditMode: boolean;}>```